pkg/syncer/syncer.go

```diff
@@ -15,6 +15,8 @@ const (
 	secretUsernameKey  = "username"
 	secretPasswordKey  = "password"
 	secretTokenKey     = "token"
+	privateKey         = "privateKey"
+	integrationId      = "integrationId"
```

Comment:
Is there a reason why its called an Integration ID? instead of App ID?

Reply:
For some reason it's called integrationId in the go-api, but let's call it appId which is what is exposed to the user in the gh ui.